### PR TITLE
Box level 4

### PR DESCRIPTION
**Pull request by s1428**

### CODE REVIEW

**Comment by s1428** [comment]
Please Review

File: index.html [hunk 1]
```diff
@@ -25,7 +26,60 @@ <h2 class="hero-heading">This is solid state </h2>
...
+                          <h3 class="third-header">Magna Arcu Feugiat</h3>
+                          <img src="images.jpg" alt="Lens">
+                        </div>
+                    </header>
+                    <hr>
+                    <p class="article-text">Lorem, ipsum dolor sit amet consectetur adipisicing Lorem ipsum dolor sitr adipisicing elit.
+                       Repudiandae nihil aut quisquam nostrum exercitationem voluptatem neque necessitatibus fugiat! Esse error aliquid beatae.
+                       Dolore quaerat, in ipsam numquam laudantium doloribus debitis,
+                       corporis aspernatur odit officiis ut blanditiis ex hic dolores fugit quisquam veniam recusandae placeat!</p>
+                       <a class="article-link" href="">Learn More</a>
```
Comment (tinkalguli) [inline index.html]:
```suggestion
                       corporis aspernatur odit officiis ut blanditiis ex hic dolores fugit quisquam veniam recusandae placeat!
                     </p>
                     <a class="article-link" href="">Learn More</a>
```

Comment (tinkalguli) [reply index.html]: Make sure the indentation of the `p` and `a` elements are right.

File: index.html [hunk 2]
```diff
@@ -25,7 +26,60 @@ <h2 class="hero-heading">This is solid state </h2>
...
+                          <h3 class="third-header">Magna Arcu Feugiat</h3>
+                          <img src="bug.jpg.crdownload" alt="Bug">
+                        </div>
+                    </header>
+                    <hr>
+                    <p class="article-text">Lorem, ipsum dolor sit amet consectetur adipisicing Lorem ipsum dolor sitr adipisicing elit.
+                       Repudiandae nihil aut quisquam nostrum exercitationem voluptatem neque necessitatibus fugiat! Esse error aliquid beatae.
+                       Dolore quaerat, in ipsam numquam laudantium doloribus debitis,
+                       corporis aspernatur odit officiis ut blanditiis ex hic dolores fugit quisquam veniam recusandae placeat!</p>
+                       <a class="article-link" href="#">Learn More</a>
```
Comment (tinkalguli) [inline index.html]: Same here, please make sure the closing `p` is exactly below the opening `p` and place the `a` element below the `p`.

File: index.html [hunk 3]
```diff
@@ -17,6 +17,7 @@ <h1>Solide State</h1>
             <a href="">Log in</a>
             <a href="">Sign Up</a>
         </nav>
+        </header>
         <main>
             <section class="Hero">
```
Comment (tinkalguli) [inline index.html]: Please don't use the `uppercase` letter in classnames.
```suggestion
            <section class="hero">
```

File: style.css [hunk 4]
```diff
@@ -14,16 +14,26 @@ body {
...
+    border-radius: 50%;
+    display: block;
+    object-fit: cover;
+}
+
 hr {
     margin: 32px auto;
-    border-color: #888;
+    border: 0;
+    border-bottom: 2px solid #999;
```
Comment (tinkalguli) [inline style.css:36]: Put the `header` styles inside `/* Header Styles */`.

File: style.css [hunk 5]
```diff
@@ -64,3 +74,46 @@ h1 {
...
+
+.blog-two {
+    background-color: #0d2e41;
+}
+
+.blog-third {
+    background-color: #1d7aaf;
+}
+
+/* Footer */
```
Comment (tinkalguli) [inline style.css]: Make sure to use a same naming convention in the comments too.

Like : 
```
/* Header */
/* Footer */
```

Or: 
```
/* Header Styles */
/* Footer Styles*/
```

File: style.css [hunk 6]
```diff
@@ -64,3 +74,46 @@ h1 {
...
+
+.footer {
+    background-color: #2E313F;
+    padding: 60px 0;
+    font-size: 20px;
+    color: #999;
+
```
Comment (tinkalguli) [inline style.css]: Please remove extra line breaks.

**Review by tinkalguli — CHANGES_REQUESTED** [review]
Please check the comments and delete the `index.review` file.

File: index.html [hunk 7]
```diff
@@ -17,15 +17,71 @@ <h1>Solide State</h1>
...
+                        <div>
+                          <h3 class="third-header">Magna Arcu Feugiat</h3>
+                          <img src="photo.jpg" alt="Camera">
+                        </div>
+                    </header>
+                    <hr>
+                    <p class="article-text">Lorem, ipsum dolor sit amet consectetur adipisicing Lorem ipsum dolor sitr adipisicing elit.
+                       Repudiandae nihil aut quisquam nostrum exercitationem voluptatem neque necessitatibus fugiat! Esse error aliquid beatae.
+                       Dolore quaerat, in ipsam numquam laudantium doloribus debitis,
+                       corporis aspernatur odit officiis ut blanditiis ex hic dolores fugit quisquam veniam recusandae placeat!</p>
```
Comment (tinkalguli) [inline index.html]: Keep the closing `p` below the opening `p`.

File: index.html [hunk 8]
```diff
@@ -17,15 +17,71 @@ <h1>Solide State</h1>
...
+                          <h3 class="third-header">Magna Arcu Feugiat</h3>
+                          <img src="images.jpg" alt="Lens">
+                        </div>
+                    </header>
+                    <hr>
+                    <p class="article-text">Lorem, ipsum dolor sit amet consectetur adipisicing Lorem ipsum dolor sitr adipisicing elit.
+                       Repudiandae nihil aut quisquam nostrum exercitationem voluptatem neque necessitatibus fugiat! Esse error aliquid beatae.
+                       Dolore quaerat, in ipsam numquam laudantium doloribus debitis,
+                       corporis aspernatur odit officiis ut blanditiis ex hic dolores fugit quisquam veniam recusandae placeat!
+                    </p>
```
Comment (tinkalguli) [inline index.html]:
```suggestion
                    <p class="article-text">
                        Lorem, ipsum dolor sit amet consectetur adipisicing Lorem ipsum dolor sitr adipisicing elit.
                        Repudiandae nihil aut quisquam nostrum exercitationem voluptatem neque necessitatibus fugiat! Esse error aliquid beatae.
                        Dolore quaerat, in ipsam numquam laudantium doloribus debitis,
                        corporis aspernatur odit officiis ut blanditiis ex hic dolores fugit quisquam veniam recusandae placeat!
                    </p>
```

File: index.html [hunk 9]
```diff
@@ -17,15 +17,71 @@ <h1>Solide State</h1>
...
+                          <h3 class="third-header">Magna Arcu Feugiat</h3>
+                          <img src="bug.jpg.crdownload" alt="Bug">
+                        </div>
+                    </header>
+                    <hr>
+                    <p class="article-text">Lorem, ipsum dolor sit amet consectetur adipisicing Lorem ipsum dolor sitr adipisicing elit.
+                       Repudiandae nihil aut quisquam nostrum exercitationem voluptatem neque necessitatibus fugiat! Esse error aliquid beatae.
+                       Dolore quaerat, in ipsam numquam laudantium doloribus debitis,
+                       corporis aspernatur odit officiis ut blanditiis ex hic dolores fugit quisquam veniam recusandae placeat!
+                    </p>
```
Comment (tinkalguli) [inline index.html]:
```suggestion
                    <p class="article-text">
                       Lorem, ipsum dolor sit amet consectetur adipisicing Lorem ipsum dolor sitr adipisicing elit.
                       Repudiandae nihil aut quisquam nostrum exercitationem voluptatem neque necessitatibus fugiat! Esse error aliquid beatae.
                       Dolore quaerat, in ipsam numquam laudantium doloribus debitis,
                       corporis aspernatur odit officiis ut blanditiis ex hic dolores fugit quisquam veniam recusandae placeat!
                    </p>
```

File: index.html [hunk 10]
```diff
@@ -17,15 +17,71 @@ <h1>Solide State</h1>
             <a href="">Log in</a>
             <a href="">Sign Up</a>
         </nav>
+        </header>
```
Comment (tinkalguli) [inline index.html]: `header` is not below the opening tag. Accordingly change the other indentations.

**Review by tinkalguli — CHANGES_REQUESTED** [review]
Please check the comments and make sure the indentation are right.

**Comment by tinkalguli** [comment]
Looks good to me.

**Pull request merged**